bc_obps/reporting/service/report_activity_save_service.py

```diff
@@ -222,4 +226,29 @@ def save_emission(
         report_emission.set_create_or_update(self.user_guid)
         EmissionCategoryMappingService.apply_emission_categories(report_source_type, report_fuel, report_emission)
 
+        self.save_methodology(report_emission, emission_data['methodology'])
+
         return report_emission
+
+    def save_methodology(
+        self,
+        report_emission: ReportEmission | None,
+        methodology_data: dict,
+    ) -> ReportMethodology:
+        json_data = exclude_keys(methodology_data, ['methodology'])
+        methodology = Methodology.objects.get(name=methodology_data['methodology'])
+
+        report_methodology_id = methodology_data.get('id')
+        report_methodology, _ = ReportMethodology.objects.update_or_create(
+            id=report_methodology_id,
+            create_defaults={
+                "methodology": methodology,
+                "json_data": json_data,
+                "report_version": self.facility_report.report_version,
+                **({"report_emission": report_emission} if report_emission else {}),
```

Comment:
No `None` case here to handle either
```suggestion
                "report_emission": report_emission,
```